src/index.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-unused-vars */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import _ from 'lodash';
+import './style.css';
+
+// Selectors
+const ul = document.querySelector('.todo-list');
+const todoContainer = document.querySelector('.todo-container');
+const clearButton = document.createElement('button');
+clearButton.classList.add('clear-button');
+clearButton.textContent = 'Clear all Completed';
+
+const todoTasks = [
+  {
+    description: 'This is my first task',
+    completed: true,
+    index: 0,
+  },
+  {
+    description: 'This is my second task',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'This is my third task',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'This is my forth task',
+    completed: true,
+    index: 3,
+  },
+  {
+    description: 'This is my fifth task',
+    completed: false,
+    index: 4,
+  },
+  {
+    description: 'This is my sixth task',
+    completed: false,
+    index: 5,
+  },
+  {
+    description: 'This is my seventh task',
+    completed: false,
+    index: 6,
+  },
+];
+
+const displayTodo = () => {
+  todoTasks.forEach((todo) => {
+    const todoItem = document.createElement('li');
+    todoItem.innerHTML = `
+          <form class="form-container">
```

Comment:
- [ ] Great job iterating over the tasks array and populating an HTML list item element for each task :clap: however The list should appear in order of the index values as required, you can do it by sorting your `todoTasks `array before using forEach method

eg: 
``` 
export const sortItems = (items) => {
  const sortedItems = items.sort((a, b) => (a.index - b.index));
  return sortedItems;
};
```

Reply:
@moise10r Thank you for your feedback. However, do I need to sort them again when already the index is already in an ascending (sorted) order? Unless I don't understand what you meant, in that case, you might want to further explain why I have to sort it again even when the index value is already sorted. Thank you.